.gitignore

```diff
@@ -1,15 +1,25 @@
-/.cache
-/.work
-/_output
-/config/
-/config
-cover.out
-/vendor
-/.vendor-new
```

Comment:
I think some or all of the above need to stay in the `.gitignore` as they're generated by the Makefile but shouldn't be checked in.